apiary.apib

```diff
@@ -870,6 +870,22 @@ Add a new company.
         + responsible_user_id: `2b12bd5d-d2d9-43ac-8755-36a1e00bd67f` (string, optional)
         + remarks: `Met at expo` (string, optional)
         + tags: prospect, expo (array[string], optional)
+        + custom_fields (array, optional)
+            + (object)
+                + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
+                + One Of
+                    + value: `092980616` (string)
+                    + value: `123` (number) - For integer, number and auto-increment fields
```

Comment:
I think we're missing `Money` too here.

Reply:
Looks like we don't allow people to pass a currency to money custom fields yet in the user interface and we just have a number here. 🤔